libraries/AP_RangeFinder/AP_RangeFinder_BLPing.cpp

```diff
@@ -17,36 +17,8 @@
 #include <GCS_MAVLink/GCS.h>
 #include "AP_RangeFinder_BLPing.h"
 
-#define BLPING_INIT_RATE_MS     1000    // initialise sensor at no more than 1hz
-#define BLPING_FRAME_HEADER1    0x42    // header first byte ('B')
-#define BLPING_FRAME_HEADER2    0x52    // header second byte ('R')
-
-#define BLPING_SRC_ID           0       // vehicle's source id
-#define BLPING_DEST_ID          1       // sensor's id
-
-#define BLPING_MSGID_ACK                    1
-#define BLPING_MSGID_NACK                   2
-#define BLPING_MSGID_SET_PING_INTERVAL      1004
-#define BLPING_MSGID_GET_DEVICE_ID          1201
-#define BLDPIN_MSGID_DISTANCE_SIMPLE        1211
-#define BLPING_MSGID_CONTINUOUS_START       1400
-
-// Protocol implemented by this sensor can be found here: https://github.com/bluerobotics/ping-protocol
-//
-// Byte     Type        Name            Description
-// --------------------------------------------------------------------------------------------------------------
-// 0        uint8_t     start1          'B'
-// 1        uint8_t     start2          'R'
-// 2-3      uint16_t    payload_length  number of bytes in payload (low byte, high byte)
-// 4-5      uint16_t    message id      message id (low byte, high byte)
-// 6        uint8_t     src_device_id   id of device sending the message
-// 7        uint8_t     dst_device_id   id of device of the intended recipient
-// 8-n      uint8_t[]   payload         message payload
-// (n+1)-(n+2)  uint16_t    checksum    the sum of all the non-checksum bytes in the message (low byte, high byte)
+const uint16_t AP_RangeFinder_BLPing::_sensor_rate_ms = 1000; // initialise sensor at no more than 1hz
```

Comment:
if this is const and internal, perhaps move it out of the class definition and make it constexpr?

Reply:
ArduPilot does not use C++11.

Reply:
@patrickelectric we actually do use C++11 (specifically gnu++11). (We even use `constexpr` already).

Reply:
@WickedShell thanks!